networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/peers/DefaultEth2Peer.java

```diff
@@ -290,9 +293,19 @@ public SafeFuture<Void> requestBlobSidecarsByRange(
                 if (updatedCount.isZero()) {
                   return SafeFuture.COMPLETE;
                 }
-                request = new BlobSidecarsByRangeRequestMessage(firstSupportedSlot, updatedCount);
+
+                request =
+                    new BlobSidecarsByRangeRequestMessage(
+                        firstSupportedSlot,
+                        updatedCount,
+                        spec.getMaxBlobsPerBlock(firstSupportedSlot.plus(updatedCount))
```

Comment:
should be technically` .plus(updatedCount).minus(1)`

Reply:
right.. good point

Reply:
put it on the other place too